src/imagetilesource.js

```diff
@@ -50,6 +50,7 @@
      * @extends OpenSeadragon.TileSource
      * @param {Object} options Options object.
      * @param {String} options.url URL of the image
+     * @param {String} options.referenceStripThumbnailUrl URL of the dedicated thumbnail image
```

Comment:
Sorry for the confusion... my suggestion, and how you've got it coded means that this new property will work for any tile source the develop gives to the viewer, and behind the scenes we will always use an image tile source in the referenceStrip, even though what's displayed in the main viewer could be any kind of tile source. That's why you're explicitly creating a fresh tile source spec with the thumbnail URL, rather than repurposing the parent tile source, which could be any type of tile source. 

Does that make sense?

By coding it this way, you've made this function more useful, since it's available for every tile source type. :)